sci-libs/silo/silo-4.11.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2
+
+DESCRIPTION="A mesh and field I/O library and scientific database"
+HOMEPAGE="https://wci.llnl.gov/simulation/computer-codes/silo"
+SRC_URI="https://wci.llnl.gov/sites/wci/files/2021-09/${P}-bsd.tgz"
+S="${WORKDIR}/${P}-bsd"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+IUSE="browser +hdf5 +silex +szip +qt5 test"
```

Comment:
Also, we don't need to add `test` to USE flags if we're not having any test-only dependencies

Reply:
Test IUSE is required for conditional test

Reply:
But where is the test conditional, I don't see the `test` USE flag being used anywhere in the ebuild?

Reply:
If I remove it and run ebuild ... test it warns me that test is not on IUSE